data-raw/watershed/floodplain_utils.R

```diff
@@ -0,0 +1,237 @@
+    modeled_length <- round(watershed_doc_vars$FR_length_modeled_mi, 1)
+  }
+
+  if (species == 'sr') {
+    rearing_length <- round(watershed_doc_vars$SR_rearing_length_mi, 1)
+    channel_area_modeled <- watershed_doc_vars$SR_channel_area_of_length_modeled_acres
+    low_grad <- round(watershed_doc_vars$SR_low_gradient_length_mi, 1)
+    high_grad <- round(watershed_doc_vars$SR_high_gradient_length_mi, 1)
+    modeled_length <- round(watershed_doc_vars$SR_length_modeled_mi, 1)
+  }
+
+  if (species == 'st') {
+    rearing_length <- round(watershed_doc_vars$ST_rearing_length_mi, 1)
+    channel_area_modeled <- watershed_doc_vars$ST_channel_area_of_length_modeled_acres
+    low_grad <- round(watershed_doc_vars$ST_low_gradient_length_mi, 1)
+    high_grad <- round(watershed_doc_vars$ST_high_gradient_length_mi, 1)
+    modeled_length <- round(watershed_doc_vars$ST_length_modeled_mi, 1)
+  }
+
+  if (watershed_method == 'full_model_nmfs') {
+
+    return(
+      glue('The entire mapped rearing extent of {rearing_length} miles was modeled',
+           ' using {model_name}. The high quality depth and high quality velocity',
+           ' ("Pref11") "BankArea" result was used as floodplain area. High quality',
+           ' velocities were assumed to be less than or equal to 0.15 meters per second,',
+           ' and high quality depths were assumed to be between 0.2 meters and 1.5 meters.')
+    )
+  }
+
+  if (watershed_method == 'full_model') {
+    return(
+      glue("The entire mapped rearing extent of {rearing_length} miles was modeled using {model_name}.",
+           " Active channel area of {channel_area_modeled} acres estimated through remote",
+           " sensing analysis was subtracted from total inundated area to get inundated floodplain area.")
+    )
+  }
+
+  if (watershed_method == 'part_model') {
+    return(
+      glue('A {modeled_length} mile portion of the entire mapped rearing extent of {rearing_length}',
+           ' miles was modeled using {model_name}. Of the entire mapped rearing extent,',
+           ' {low_grad} miles were classified as low gradient and {high_grad} miles were classified',
+           ' as high gradient based on a geomorphic analysis of long profile slopes and valley widths.',
+           ' Floodplain area per unit length was determined for the modeled extent and applied to',
+           ' determine areas for the non-modeled extent. A scaling factor of {high_grad_factor} was',
+           ' applied to the area per unit length for the high gradient extent. No',
+           ' scaling factor was applied to the low gradient extent.')
+    )
+  }
+
+  if (str_detect(watershed_method, 'scaled')) {
+    proxies <- c('Deer Creek', 'Cottonwood Creek', 'Tuolumne River')
+    names(proxies) <- c('dc', 'cc', 'tr')
+
+    proxy_ws <- proxies[str_remove(watershed_method, 'scaled_')]
+
+    return(
+      glue("No watershed specific hydraulic modeling was available for {watershed_name}.",
```

Comment:
nit picky comment - single quotes are used throughout this script, you used double. plz make effort to match style of existing code within a script

Reply:
done